README.md

```diff
@@ -1,5 +1,5 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_task-move-spider-DOM/)
+    - [DEMO LINK](https://Inna_Pohrebna.github.io/js_task-move-spider-DOM/)
```

Comment:
there is no gh-pages for the moment, check in repo settings that correct branch is chosen

https://user-images.githubusercontent.com/38065883/120308878-3fdde600-c2dd-11eb-9217-8485440e64d7.mov

Reply:
Sorry, but I can't understand. I did npm run deploy and I saw that my task have gp-pages. Can you explain me please What I have to do? 
![spider](https://user-images.githubusercontent.com/79695381/120311434-24c0a580-c2e0-11eb-88aa-bc180d9014ea.png)

Reply:
In your repository setting change branch to built from to `develop` and back to `gh-pages`